pkg/nmstatectl/converter.go

```diff
@@ -6,11 +6,19 @@ import (
 	"github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate.io/v1"
 	nmstatev1 "github.com/nmstate/kubernetes-nmstate/pkg/client/clientset/versioned/typed/nmstate.io/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os"
 	"os/exec"
 )
 
 const nmstateCommand = "nmstatectl"
 
+func Check() error {
+	cmd := exec.Command(nmstateCommand, "show")
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	return cmd.Run()
```

Comment:
AFAIK this returns only error code. We are more interested in stderr, that should be passed as error to the caller. Please correct me if that is wrong. It would be great if you could try it and show the output.

Reply:
Ok, used the CombinedOutput so we wrap the std[out,err] into an error.